web/cobrands/tfl/base.scss

```diff
@@ -183,6 +188,12 @@ ol.big-numbers>li {
     }
 }
 
+.item-list--reports li > a,
+#key-tools a,
+.problem-back {
+    text-decoration: none;
```

Comment:
Fine for now, I wonder if this should actually be in base so that it works for any cobrand that adds text-decoration.